src/utils/payments/pawapay/types.ts

```
@@ -0,0 +1,69 @@
+  msisdn: string;
+  statementDescription?: string;
+  metadata?: Array<{
+    fieldName: string;
+    fieldValue: string;
+    isPII?: boolean;
+  }>;
+  reason?: string;
+  correspondent?: string;
+}
+
+interface TransactionResponse {
```

ESLint

[error] Use a `type` instead of an `interface`.

(ts/consistent-type-definitions)